linetools/isgm/utils.py

```diff
@@ -325,6 +326,55 @@ def complist_from_table(table):
     return complist
 
 
+def table_from_complist(complist):
+    """
+    Returns a QTable from an input list of AbsComponents. It only
```

Comment:
You are right; I find particularly annoying that vstack does not work properly with QTables. Will use Tables as default then. Still, there will be one place where QTable is more useful (e.g. to access entry values as quantity objects), but will be handled internally in the corresponding function; the user will always get a Table object out.